docs/videos/healthcare-and-life-sciences/README.md

```diff
@@ -78,61 +104,60 @@ storage intact and b) you can build software before you deploy your cluster.
    make
    ```
 
-1. Generate the deployment folder
-
-   Starting with the command below, populate the `project_id` and the three
-   bucket name variables with unique bucket names for your deployment. The three
-   buckets will be created as part of the deployment and used as follows:
-
-   - **bucket_name_inputs:** name for GCS bucket that will be used as a library
-     of pre-loaded static inputs (protein library, benchmarks, examples).
-   - **bucket_name_outputs:** name for GCS bucket that will be used to retain,
-     share, and utilize results in downstream analysis.
-   - **bucket_name_software:** name for GCS bucket where users can upload
-     software to be utilized by the cluster. See *Upload VMD tarball* step
-     below.
-
-   The `homefs_server_ip` and `appsfs_server_ip` will be populated after the
-   setup stage is complete. For now we will populate with a dummy value to allow
-   HPC Toolkit to run.
+1. Generate the deployment folder after replacing `<project>` with the project id.
 
    ```bash
-   ./ghpc create docs/videos/healthcare-and-life-sciences/hcls-blueprint.yaml -w \
-       --vars project_id=<project> \
-       --vars bucket_name_inputs=<input_bucket_name> \
-       --vars bucket_name_outputs=<output_bucket_name> \
-       --vars bucket_name_software=<software_bucket_name> \
-       --vars homefs_server_ip=1.1.1.1 \
-       --vars appsfs_server_ip=1.1.1.1
+   ./ghpc create docs/videos/healthcare-and-life-sciences/hcls-blueprint.yaml -w --vars project_id=<project>
    ```
 
 1. Deploy the `enable_apis` group
 
-   Call the following terraform commands to deploy the `enable_apis` deployment
-   group.
+   Call the following ghpc command to deploy the the hcls blueprint.
 
    ```bash
-   terraform -chdir=hcls-01/enable_apis init && terraform -chdir=hcls-01/enable_apis apply
+   ./ghpc deploy hcls-01
    ```
 
+   This will prompt you to **display**, **apply**, **stop**, or **continue**
+   without applying the `enable_apis` group. Select apply.
+
    This will ensure that all of the needed apis are enabled before deploying the
    cluster.
 
+   > [!NOTE]
+   > This ghpc command will run through 4 groups (`enable_apis`, `setup`,
+   > `software_installation`, and `cluster`) and prompt you to apply each one.
+   > If the command is cancelled or exited by accident before finishing, it can
+   > be rerun to continue deploying the blueprint.
+
 1. Deploy the `setup` group
 
+   The next `ghpc` prompt will ask you to **display**, **apply**, **stop**, or
+   **continue** without applying the `setup` group. Select 'apply'.
+
    This group will create a network and file systems to be used by the cluster.
 
-   Call the following terraform commands to deploy the `setup` deployment
-   group.
+   > [!NOTE]
```

Comment:
nit: I would make this one a warning so people moving fast don't miss it. Personal opinion formed while running through the instructions.